test-utils/client/src/lib.rs

```diff
@@ -330,59 +311,6 @@ impl std::fmt::Display for RpcTransactionError {
 	}
 }
 
-/// An extension trait for `RpcHandlers`.
-pub trait RpcHandlersExt {
```

Comment:
Why was this removed? This is used in integration tests in Cumulus

Reply:
As long as you fix the code in Polkadot/Cumulus. We don't need to bring it back.

Reply:
We'll take a look at cumulus tests then, thanks for pointing it out.